ArmaReforgerServerTool/Managers/FileIOManager.cs

```diff
@@ -113,13 +115,29 @@ public void ReadModsDatabase()
         /// </summary>
         public static void SaveConfigurationToFile()
         {
+            FileIOManager instance = GetInstance();
             using SaveFileDialog sfd = new();
-            sfd.InitialDirectory = Environment.SpecialFolder.UserProfile.ToString();
+
+            // Use lastUsedDirectory if available
+            if (!string.IsNullOrEmpty(instance.lastUsedDirectory))
+            {
+                sfd.InitialDirectory = instance.lastUsedDirectory;
```

Comment:
instead of calling this instance in here which I think might be a bit weird, have a getter which means we can, for static methods go `FileIOManager.GetInstance().GetLastUsedDirectory()`. This would make it consistent with the rest of the codebase too